views/jsconfig.handlebars

```diff
@@ -1,8 +1,16 @@
+{{!-- This file should be es5 only --}}
+var isIE = navigator.userAgent.toLowerCase().indexOf('msie') > -1 ||
+    !!navigator.userAgent.toLowerCase().match(/trident\/7\./);
```

Comment:
Instead of a double negation on `.match()`, we could possibly just use `.test()`:

```js
/trident\/7\./i.test(navigator.userAgent);
```